koch.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
```

Comment:
Not saying is a good idea, but the only way I see not having this file without being driven mad is have Koch check something on startup, then either rebuild the real Koch or use the real Koch to do the work... so a proxy in which case we're basically back to this script, except in nimskull, and only difference is because we don't need a batch file for Windows?

I'm not sure that's all that great an idea, I'd lean script because that can come later of it becomes a pain.